stock_picking_package_preparation/views/stock_picking_package_preparation_view.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="0">
+  <record id="stock_picking_package_preparation_tree" model="ir.ui.view">
```

Comment:
@eLBati need XML indentation

Reply:
@Nikul-Chaudhary this file uses 2 spaces indentation. Is this not recommended?

Reply:
> @Nikul-Chaudhary this file uses 2 spaces indentation. Is this not recommended?

Yes need 4, Look as https://github.com/OCA/maintainer-tools/blob/master/template/module/views/model_name_view.xml#L8

Reply:
@Nikul-Chaudhary that file is a template, but https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst doesn't talk about XML indentation.
Consider that this file is there since several versions, I am not sure we should change it now.

Reply:
@Nikul-Chaudhary Indeed, this is not blocking for me. Please update yor review